packages/@okta/vuepress-site/docs/reference/api/user-types/index.md

```diff
@@ -212,19 +212,19 @@ curl -s -XPOST -H "Content-Type: application/json" -H "Authorization: SSWS ${api
 
 <ApiOperation method="POST" url="/api/v1/meta/types/user/{typeId}" />
 
-Updates an existing User Type. A PUT is a full replace operation; a POST is a partial update. Only the `displayName` and `description` elements can be changed; the `name` of an existing User Type cannot be changed.
+Updates an existing User Type. A PUT is a full replace operation; a POST is a partial update. Only the `displayName` and `description` elements can be changed; the `name` of an existing User Type can't be changed.
```

Comment:
avoid semi-colons in sentences and make present tense.

Updates an existing User Type. A PUT is a full replace operation. A POST is a partial update. You can only change the `displayName` and `description` elements. You can't change the `name` of an existing User Type.